deployment/ansible/roles/nyc-trees.tiler/tasks/dependencies.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Install tiler javascript dependencies
+  command: npm install
```

Comment:
It looks like NPM drops privileges to use `gyp` during this process, which generated a bunch of warnings and changes the directories `gyp` uses. Adding `--unsafe-perm` here alleviates that.